frontend/src/i18n/declaration.ts

```diff
@@ -300,6 +300,7 @@ export enum I18nKey {
   STATUS$ERROR_LLM_AUTHENTICATION = "STATUS$ERROR_LLM_AUTHENTICATION",
   STATUS$ERROR_LLM_SERVICE_UNAVAILABLE = "STATUS$ERROR_LLM_SERVICE_UNAVAILABLE",
   STATUS$ERROR_LLM_INTERNAL_SERVER_ERROR = "STATUS$ERROR_LLM_INTERNAL_SERVER_ERROR",
+  STATUS$ERROR_LLM_INTERNAL_SERVER_ERROR_MESSAGE = "STATUS$ERROR_LLM_INTERNAL_SERVER_ERROR_MESSAGE",
```

Comment:
We generally don't suffix with `_MESSAGE`...

something feels off here. Why do we have two different keys?

Reply:
One for the red text and one for the blue one
<img width="842" alt="image" src="https://github.com/user-attachments/assets/1eb9ff10-b250-4aca-8d90-6e9e31864442" />

Otherwise user ends up having the same error message twice which IMO is even more confusing